src/Tizen.NUI/src/public/BaseComponents/VectorGraphics/Shape.cs

```diff
@@ -399,6 +400,40 @@ public bool AddCubicTo(float controlPoint1X, float controlPoint1Y, float control
             return ret;
         }
 
+
+        /// <summary>
+        /// Appends a given sub-path to the path.
+        /// The current point value is set to the last point from the sub-path.
+        /// @note The interface is designed for optimal path setting if the caller has a completed path commands already.
+        /// </summary>
+        /// <param name="pathCommands">The command object that contain sub-path information. (This command information is copied internally.)</param>
+        /// <exception cref="ArgumentNullException"> Thrown when pathCommands is null. </exception>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public void AddPath(PathCommands pathCommands)
+        {
+            if (pathCommands == null)
+            {
+                throw new ArgumentNullException(nameof(pathCommands));
+            }
+
+            PathCommandType[] commands = null;
+            if (pathCommands.Commands is PathCommandType[] commandArray)
+            {
+                commands = commandArray;
+            }
+            commands = pathCommands.Commands.ToArray();    
+
+            float[] points = null;            
+            if (pathCommands.Points is float[] pointArray)
+            {
+                points = pointArray;
+            }
+            points = pathCommands.Points.ToArray();    
+            
+            Interop.Shape.AddPath(BaseHandle.getCPtr(this), commands, (uint)pathCommands.Commands.Count(), points, (uint)pathCommands.Points.Count());
```

Comment:
이미 변환된 배열이 있고, C#의 배열 타입은 Length property가 있으므로 이걸 사용하는게 더 나을듯합니다.
```suggestion
            Interop.Shape.AddPath(BaseHandle.getCPtr(this), commands, (uint)commands.Length, points, (uint)points.Length);
```